### PR TITLE
Update enable-controlled-folders.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to navigation in endpoint manager to enable controlled folder access

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4981

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 1]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
 > If the feature is set to **Audit mode** with any of those tools, the Windows Security app will show the state as **Off**.
 > If you are protecting user profile data, we recommend that the user profile should be on the default Windows installation drive.
 
-## Intune
+## Endpoint Manager
 
-1. Sign in to the [Azure portal](https://portal.azure.com) and open Intune.
+1. Sign in to the [Endpoint Manager](https://endpoint.microsoft.com) and open Endpoint Security.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
1. Sign in to the [Endpoint Manager](https://endpoint.microsoft.com) and open **Endpoint Security**.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 2]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
 
-2. Go to **Device configuration** > **Profiles** > **Create profile**.
+2. Go to **Attack Surface Reduction** > **Policy**.
 
-3. Name the profile, choose **Windows 10 and later** and **Endpoint protection**. <br/> ![Create endpoint protection profile](/microsoft-365/security/defender-endpoint/images/create-endpoint-protection-profile) <br/>
+3. Select the Plaform , choose **Windows 10 and later** and Select the profile **Attack Surface Reduction rules** > Click **Create**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
3. Select **Platform**, choose **Windows 10 and later**, and select the profile **Attack Surface Reduction rules** > **Create**.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 3]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
 
-4. Go to **Configure** > **Windows Defender Exploit Guard** > **Controlled folder access** > **Enable**.
+4.  **Name** the Policy and add **Description** > Click **Next**
 
-5. Type the path to each application that has access to protected folders and the path to any additional folder that needs protection. Select **Add**.<br/> ![Enable controlled folder access in Intune](/microsoft-365/security/defender-endpoint/images/enable-cfa-intune)<br/>
+5.  Scroll down to the bottom and Select **Enable Folder Protection** Drop down and Choose **Enable**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
5.  Scroll down to the bottom, select the **Enable Folder Protection** drop-down, and choose **Enable**.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 4]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
 
-   > [!NOTE]
-   > Wilcard is supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.
+6.  Select **List of additional folders that need to be protected** and **Add** the folders that needs to be protected.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
6.  Select **List of additional folders that need to be protected** and add the folders that need to be protected.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 5]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
+
+7.  Select **List of apps that have access to protected folders** and **Add** the apps that have access to protected folders.
+
+8.  Select **Exclude files and paths from attack surface reduction rules** and **Add** the Files and paths that needs to be excluded from Attack surface Reduction rules.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
8.  Select **Exclude files and paths from attack surface reduction rules** and add the files and paths that need to be excluded from attack surface reduction rules.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 6]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
+1. Sign in to the [Endpoint Manager](https://endpoint.microsoft.com) and open Endpoint Security.
 
-2. Go to **Device configuration** > **Profiles** > **Create profile**.
+2. Go to **Attack Surface Reduction** > **Policy**.
 
-3. Name the profile, choose **Windows 10 and later** and **Endpoint protection**. <br/> ![Create endpoint protection profile](/microsoft-365/security/defender-endpoint/images/create-endpoint-protection-profile) <br/>
+3. Select the Plaform , choose **Windows 10 and later** and Select the profile **Attack Surface Reduction rules** > Click **Create**
 
-4. Go to **Configure** > **Windows Defender Exploit Guard** > **Controlled folder access** > **Enable**.
+4.  **Name** the Policy and add **Description** > Click **Next**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
4.  Name the policy and add a description. Select **Next**.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 7]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
-   > [!NOTE]
-   > Wilcard is supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.
+6.  Select **List of additional folders that need to be protected** and **Add** the folders that needs to be protected.
+
+7.  Select **List of apps that have access to protected folders** and **Add** the apps that have access to protected folders.
+
+8.  Select **Exclude files and paths from attack surface reduction rules** and **Add** the Files and paths that needs to be excluded from Attack surface Reduction rules.
 
-6. Select **OK** to save each open blade and **Create**.
+9.  Select the profile **Assignments**, assign to **All Users & All Devices**, and **Save**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
9.  Select the profile **Assignments**, assign to **All Users & All Devices**, and select **Save**.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 8]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
 
-7. Select the profile **Assignments**, assign to **All Users & All Devices**, and **Save**.
+10.  Select **Next** to save each open blade and **Create**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
10.  Select **Next** to save each open blade and then **Create**.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 9]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
+
+   > [!NOTE]
+   > Wilcard is supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
   > Wildcards are supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.
```

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 10]
```diff
@@ -59,24 +59,30 @@ For more information about disabling local list merging, see [Prevent or allow u
...
+4.  **Name** the Policy and add **Description** > Click **Next**
 
-5. Type the path to each application that has access to protected folders and the path to any additional folder that needs protection. Select **Add**.<br/> ![Enable controlled folder access in Intune](/microsoft-365/security/defender-endpoint/images/enable-cfa-intune)<br/>
+5.  Scroll down to the bottom and Select **Enable Folder Protection** Drop down and Choose **Enable**
 
-   > [!NOTE]
-   > Wilcard is supported for applications, but not for folders. Subfolders are not protected. Allowed apps will continue to trigger events until they are restarted.
+6.  Select **List of additional folders that need to be protected** and **Add** the folders that needs to be protected.
+
+7.  Select **List of apps that have access to protected folders** and **Add** the apps that have access to protected folders.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
7.  Select **List of apps that have access to protected folders** and add the apps that have access to protected folders.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder.

**Pull request merged**